prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -102,6 +103,17 @@ class FHIRConverter(
         val format = Report.getFormatFromBlobURL(queueMessage.blobURL)
         logger.trace("Processing $format data for FHIR conversion.")
 
+        // This line is a workaround for a defect in the hapi-fhir library
+        // Specifically https://github.com/hapifhir/hapi-fhir/blob/b555498c9b7824af67b219e5b7b85f7992aec991/hapi-fhir-serviceloaders/hapi-fhir-caching-api/src/main/java/ca/uhn/fhir/sl/cache/CacheFactory.java#L32
+        // which creates a static instance of ServiceLoader which the documentation indicates is not safe to use in a
+        // concurrent setting https://arc.net/l/quote/hauavetq.  See also this closed issue https://github.com/jakartaee/jsonp-api/issues/26#issuecomment-364844610
+        // for someone requesting a similar change in another library and the reasoning why it can't be done that way
+        //
+        // This line exists so that FhirPathUtils (an object) is instantiated before any of the multi-threaded code run
+        // (kotlin objects are instantiated at first access https://arc.net/l/quote/tbvpqnlh)
+        // TODO: https://github.com/CDCgov/prime-reportstream/issues/14287
+        FhirPathUtils
```

Comment:
do you have to call something from within the object or is just referencing the object enough?

Reply:
Yep referencing it is enough